eclair-core/src/main/scala/fr/acinq/eclair/db/NetworkDb.scala

```diff
@@ -49,7 +50,9 @@ trait NetworkDb {
 
   def listChannelUpdates(): Seq[ChannelUpdate]
 
-  def addToPruned(shortChannelId: ShortChannelId)
+  def addToPruned(shortChannelId: ShortChannelId): Unit = addToPruned(Seq(shortChannelId))
```

Comment:
Why did you keep `def addToPruned(shortChannelId: ShortChannelId)`?

Reply:
No real reason, I've removed it.